WearTilesKotlin/app/src/main/java/com/example/wear/tiles/fitness/FitnessTileService.kt

```diff
@@ -118,18 +118,25 @@ class FitnessTileService : TileProviderService() {
                     .addContent(
                         Text.builder()
                             .setText(goalProgress.current.toString())
-                            .setFontStyle(
-                                FontStyle.builder().setSize(sp(44f)).build()
-                            )
                             .setFontStyle(FontStyles.display2(deviceParameters))
                     )
                     .addContent(
                         Text.builder()
-                            .setText(resources.getString(R.string.goal, goalProgress.goal))
-                            .setFontStyle(
-                                FontStyle.builder().setSize(sp(44f)).build()
-                            )
+                            .setText(getString(R.string.tile_fitness_goal, goalProgress.goal))
                             .setFontStyle(FontStyles.title3(deviceParameters))
                     )
             )
+            .setModifiers(
+                ModifiersBuilders.Modifiers.builder()
```

Comment:
Nit: import Modifiers and Semantics directly to keep this consistent with the other builders.

Reply:
Updated in the latest commit!